frontend/packages/noobaa-storage-plugin/src/plugin.ts

```diff
@@ -92,6 +123,25 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [NOOBAA_FLAG],
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: [
+        `/k8s/ns/:ns/${referenceForModel(ClusterServiceVersionModel)}/:appName/${referenceForModel(
```

Comment:
Would be better if you could extract `referenceForModel(ClusterServiceVersionModel)` once in  a variable and use everywhere.

Reply:
Keeping it this way IMO keeps it more readable. It conveys that we are using the GVK name of the CSVModel.